src/Resource/GenericResource.php

```diff
@@ -125,6 +125,18 @@ public function listChildren()
     /**
      * {@inheritdoc}
      */
+    public function getStack()
+    {
+        if (!$this->getRepository()) {
+            throw ResourceNotFoundException::forPath($this->getRepositoryPath());
```

Comment:
I don't know ... It's just the same that is thrown on other methods like this one.